db/schema.rb

```diff
@@ -0,0 +1,44 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_05_08_143317) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "reservations", force: :cascade do |t|
+    t.integer "user_id"
+    t.integer "tutor_id"
+    t.date "date"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "tutors", force: :cascade do |t|
+    t.string "name"
+    t.string "image"
+    t.string "speciality"
+    t.string "bio"
+    t.bigint "price"
```

Comment:
- Let's consider the price for a tutor hour, maybe this could reach 2 numbers and 2 decimals, maybe 3 or 4 but no more than that (if we talked about dollars, maybe in another coin could be 7 or 8), in any way, a big integer is a lot for this, why don't use decimal or similar? (again, my POV) 🐱

Reply:
Hello @vichuge thank you for the suggestion, its implemented